src/pages/organize/[orgId]/people/[personId]/index.tsx

```diff
@@ -123,12 +126,14 @@ const PersonProfilePage: PageWithLayout<PersonPageProps> = (props) => {
             )}
           </ZetkinQuery>
         </Grid>
-        <Grid item lg={4} xs={12}>
-          <PersonJourneysCard
-            orgId={orgId as string}
-            personId={personId as string}
-          />
-        </Grid>
+        {journeys && journeys?.length > 0 && (
```

Comment:
It should be possible to just have `journeys?.length > 0` here. Check out MDN on how the [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) works.

Reply:
That's unexpected! Probably because it's being used in a comparison, where `journeys?.length` is evaluated to `undefined` if `journeys` is nullish, and `undefined > 0` makes no sense.

I would change it to one of these:

### Manual chaining (without optional chaining operator)
```tsx
{journeys && journeys.length > 0 && (
```
In this case, you don't need the optional chaining operator because you're already doing the same thing manually by checking that `journeys` is truthy before comparing `journeys.length`.

### Optional chaining operator and truthy length
```tsx
// Check that journeys.length exists and is truthy
{journeys?.length && (
```

In this case, we're not comparing `length` to a number, just checking that it's truthy, which is synonymous to `>  0`. Because we're not comparing it to a number, we don't care about it's type, as long as it's truthy. And since we don't care about it's type, it's not a problem that `journeys?.length` will evaluate to a number when `journeys` is set, or `undefined` if `journeys` is nullish.

Reply:
Thank you for clear explanation! I think the second option is more clear to read, and makes more sense to use! The first option is what I changed it to when I looked at the code again, bc i also realised that the optional chaining was unneccessary. But I will change to option 2 because it's easier on the eyes imo.